docs/customer-portal/cp-dashboard.md

```diff
@@ -12,20 +12,36 @@ The **Customer Portal** provides access to configurations, real-time testing and
 
 + **Channel Count** Total Live Channels presently in use and total configured Channels.
 
-+ **Account Balance** Displays the current account balance and a link to make a payment using a credit card or PayPal.
++ **Account Balance** Displays the current account balance and a link to make a payment using a credit card or PayPal. [Click here](https://docs.connexcs.com/customer-portal/cp-payment/) to know more about **Payments**.
 
 + **Account** Use this to view your Company Profile, view and update your personal Profile, Change Password or Language, or Log Out.
 
+### Setup Personal Profile
+
+To setup, click your name in the upper right corner and select `My Profile`.
+
+=== "Basic"
+
+    View account details such as **Name**, **Role**, **Email**, **Department**, **Phone**,  **Mobile** and **Skype**. 
+    
+    **You can only edit the Mobile field. To change email or name, contact ConnexCS Support.**
+    
+    **Two Factor Authentication**: This adds additional security against password guessing and theft, including brute force attacks (details [**here**](https://www.ncsc.gov.uk/guidance/multi-factor-authentication-online-services)). 
+    
+    To configure this on ConnexCS, click `Enable` and follow prompts for QR code. You must install a 2FA/TFA app on your phone. 
+    
+    <img src= "/customer-portal/img/profile.png" width= "500">
+
 ## Graphs
 
 Below is the Current Customer account usage on four default **Graphs**.
 
 |Graph|Description |
-|:------------|:-------------------------------------------------|
-|**Live Channels** |Details of active channel usage|
-|**ASR** |You can view Answer Seizure Ratio as a Total, or isolate traffic on a per-carrier basis (click on a customer to remove or add it from the graph); if some carriers are lower than others, this may create an issue|
+|:----|:-----------|
+|**Live Channels**|Details of active channel usage|
+|**ASR**|You can view Answer Seizure Ratio as a Total, or isolate traffic on a per-carrier basis (click on a customer to remove or add it from the graph); if some carriers are lower than others, this may create an issue|
 |**ACD**|You can view the total average call duration or isolate traffic by carrier (click on a customer to remove or add it to the graph). A lower ACD on one carrier is indicative of potential line quality issues|
-|**DTMF** |DTMF (Dual Tone Modulated Frequency) percentages|
+|**DTMF**|DTMF (Dual Tone Modulated Frequency) percentages|
 
 ## Summary

```

Comment:
In summary remove Generate Invoice as its CP and we don't have an option to Generate Invoice in CP

Reply:
removed